app/uk/gov/hmrc/fileupload/ApplicationLoader.scala

```diff
@@ -93,6 +95,16 @@ class ApplicationModule(context: Context) extends BuiltInComponentsFromContext(c
     envelopeRepository.update() _
   }
 
+  def envelopeConstraintsConfiguration(config: Configuration) = {
+    EnvelopeConstraints(config.getInt("envelopeDefaultConstraints.maxNumFiles").getOrElse(throwRuntimeException("maxNumFiles")),
+                               config.getString("envelopeDefaultConstraints.maxSize").getOrElse(throwRuntimeException("maxSize")),
+                               config.getString("envelopeDefaultConstraints.maxSizePerItem").getOrElse(throwRuntimeException("maxSizePerItem")))
+  }
+
+  def throwRuntimeException(key: String) = {
```

Comment:
those default constraints will stored in application.conf, if there is many in the conf, the app will fail at start point.